Sources/SwiftPlot/PlotStyleHelpers.swift

```diff
@@ -67,3 +67,51 @@ struct Text : Annotation {
         self.location = location
     }
 }
+
+struct Arrow : Annotation {
+    public var color = Color.black
+    public var start = Point(0.0, 0.0)
+    public var end = Point(0.0, 0.0)
+    public var strokeWidth: Float = 10
+    public var headLength: Float = 10
+    public var headAngle: Float = 20
+    public func draw(renderer: Renderer) {
+        // Draws arrow body.
+
+        //TODO: scale points to be plotted according to plot size
+        
+        renderer.drawLine(startPoint: start,
+                          endPoint: end,
+                          strokeWidth: strokeWidth,
```

Comment:
Changing "strokeWidth" seems to have no effect on the output.

Reply:
You have just stumbled into a bug in CPPAGGRenderer! If you take a look at the output for SVG you can see your changes are taking effect. If you want you can fix it by going to CPPAGGRenderer.cpp and write `stroke.width(thickness);` in line 350.

Reply:
Thank you for noticing this! I'll create an issue for this and a PR to fix it.

Reply:
Update: see #73